### ISSUE TITLE
os::linux::local::plugin --mode traffic Ignore administative down interfaces

### ISSUE THREAD

**Issue by e3k** [centreon/centreon-plugins#2911]
hi is it possible with linux local plugin in mode traffic to ignore the state of interfaces which are administratively put down?

i have found only the option:

--critical-status
            Set critical threshold for status (Default: '%{status} ne
            "RU"'). Can used special variables like: %{status}, %{display}

but no real doc that would explain how to achieve ignoring of the admin down interfaces.

is it possible with the current plugin or would this be a new feature request?

**Comment by garnier-quentin** [comment]
That how it works:
```
R = RUNNING|LOWER_UP
U = UP
```

Maybe we can add following capabilites to be more precise (I'm not sure how it should work now)
```
LOWER_UP = Driver signals L1 up (since Linux 2.6.17)
RUNNING = Resources allocated
UP = Interface is running
NO-CARRIER =  IF (UP is set) AND (RUNNING is not set)
```

**Comment by lucie-dubrunfaut** [comment]
Hello :)

This issue seems particularly old, but @e3k can you tell us if Quentin's answer helps you to better understand the `--critical-status` option and how to use it to meet your needs ?

**Comment by e3k** [comment]
Hi Lucie, it has been a while... Currently i  have no more access to the system but if i recall it correctly the problem was that "admin down" devices were considered as a critical status. I opened this one maybe as a question "how to ignore admin down devices". But it also might be feature request.

**Comment by lucie-dubrunfaut** [comment]
Yeah it's true that the question is very old ^^, but I would like to be able to determine if the plugin as it is designed can answer it. I'm not sure I'll be able to correctly make the link between "admin down" and the statuses explained by Quentin above. Particularly without having been able to test myself on "admin down" data sample to see/check the value of the [$state variable](https://github.com/centreon/centreon-plugins/blob/develop/src/os/linux/local/mode/traffic.pm#L222).

**Comment by lucie-tirand** [comment]
Hello :)

Let me close this issue, but if it becomes relevant again or if anyone else has the same, feel free to reopen this topic.
